templates/zerver/app/compose.html

```diff
@@ -15,33 +15,19 @@
                     <span class="alert-draft pull-left">{{ _('Saved as draft') }}</span>
                 </span>
                 <span class="new_message_button">
-                    <button type="button" class="button small rounded compose_mobile_button"
-                      id="left_bar_compose_mobile_button_big"
-                      title="{{ _('New message') }} (c)">
-                        <span>+</span>
+                    <button type="button" class="button small rounded compose_new_topic_button"
```

Comment:
We should use id here instead of class. `$('#compose_new_topic_button')` fits the semantics better.